api/src/main/scala/org/virtuslab/ideprobe/protocol/Endpoints.scala

```diff
@@ -29,6 +29,11 @@ object Endpoints {
   val ListOpenProjects = Request[Unit, Seq[ProjectRef]]("projects/all")
   val Messages = Request[Unit, Seq[IdeMessage]]("messages")
   val ModuleSdk = Request[ModuleRef, Option[String]]("module/sdk")
+  val ModuleDependencies = Request[ModuleRef, Seq[ModuleRef]]("module/dependencies")
+  val ModuleSources = Request[ModuleRef, Seq[Path]]("module/sources")
```

Comment:
Maybe we could just have one endpoint for content roots of module? Also content roots are already in basic project model, maybe it could be extended with types

Reply:
Now, the contents root in the project model are a map of ContentRoot -> Set of paths